crates/rolldown_testing/Cargo.toml

```diff
@@ -22,3 +22,8 @@ rolldown_common = { workspace = true }
 schemars        = { workspace = true }
 serde           = { workspace = true, features = ["derive"] }
 serde_json      = { workspace = true }
+
+[build-dependencies]
+rolldown_testing_config = { workspace = true }
```

Comment:
Wonder if we could point to `rolldown_testing` itself. if it could, we don't need to introduce a new crate anymore.